bindings/ergo-lib-c/cbindgen.toml

```diff
@@ -141,3 +141,11 @@ crates = ["ergo-lib-c"]
 all_features = false
 default_features = true
 features = []
+
+
+[ptr]
+# An optional string to decorate all pointers that are
+# required to be non null. Nullability is inferred from the Rust type: `&T`,
+# `&mut T` and `NonNull<T>` all require a valid pointer value.
+non_null_attribute = "_Nonnull"
```

Comment:
I looked up this feature, very nice. I wonder if it's worth refactoring the rest of the codebase to use `NonNull`.

Reply:
Yes, I found it when I was constructing `CompletionCallback` in Swift - https://github.com/ergoplatform/sigma-rust/blob/92fb1e3a3b7e440734bb08b7b1f1502c8fb646b3/bindings/ergo-lib-c-core/src/rest/api/callback.rs#L9-L19
I believe it makes sense mainly in the struct properties. Do we have any, besides the new `*Callback`?

Reply:
I just checked and the only places we have it are for `Return[Num|Bool|Option]`. These structs have the `error :: ErrorPtr` field which is supposed to be `null` if there's a valid value. So looks like we don't need to make further changes elsewhere.